go.mod

```diff
@@ -32,7 +32,7 @@ require (
 	github.com/pingcap/pd v2.1.0-rc.4+incompatible
 	github.com/pingcap/tidb-tools v0.0.0-20181112132202-4860a0d5de03
 	github.com/pingcap/tipb v0.0.0-20181012112600-11e33c750323
-	github.com/pkg/errors v0.8.0 // indirect
+	github.com/pkg/errors v0.8.0
```

Comment:
What is this change for?

Reply:
Because I explicitly imported "pkg/errors", already changed to "pingcap/errors"